//package com.sincetimes.redwar.game.comnunicate.protocol;
message RceUpdateStarsBookmarks
{              
	optional int32 planetId = 1;
	optional int32 coordX = 2;
	optional int32 starName = 3;
	optional int32 coordY = 4;
	optional int32 starType = 5;
	optional int32 starId = 6;
	optional string action = 7;//addBookmark
}